include/bluetooth/services/hrs_client.h

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef BT_HRS_CLIENT_H_
+#define BT_HRS_CLIENT_H_
+
+/**
+ * @file
+ * @defgroup bt_hrs_client Heart Rate Service Client
+ * @{
+ * @brief Heart Rate (HR) Service Client API.
+ */
+
+#include <bluetooth/conn.h>
+#include <bluetooth/gatt_dm.h>
+#include <bluetooth/gatt.h>
+
+#include <sys/atomic.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**@brief Heart Rate Service error codes
+ *
+ * This service defines the following Attribute Protocol Application error codes.
+ */
+enum bt_hrs_client_error {
+	/** Control Point value not supported. */
+	BT_HRS_CLIENT_ERROR_CP_NOT_SUPPORTED = 0x80
+};
```

Comment:
I think that the user does not need to know application error codes. This definition can be moved to implementation files.

Reply:
Actually this error code can be passed to the application as argument of write complete callback when you write wrong value to the CP characteristic

Reply:
I see, thanks